src/Uno.UI/UI/Xaml/Window.wasm.cs

```diff
@@ -197,6 +199,23 @@ private static Window InternalGetCurrentWindow()
 			return _current;
 		}
 
+		internal void UpdateRootAttributes()
+		{
+			if (_window == null)
+			{
+				throw new InvalidOperationException("Internal window root is not yet set.");
+			}
+
+			if (FeatureConfiguration.ButtonBase.UseHandCursor)
+			{
+				_window.SetAttribute("data-buttonbase-usehandcursor", "true");
+			}
```

Comment:
Since this is the default, can we avoid calling this on startup and set it as a default directly in TypeScript?

Reply:
So upon further inspection this is unfortunately not possible, as the `_window` here is not the "HTML window", but in fact a `Grid` which is generated when the `Window.Content` is first set, so I think setting the attribute in TS would be unnecessarily hacky in this case.